<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="75d6218e-0a16-42a6-a42a-de2a521d0d17" name="monash@EED-2710-DEV 2010-02-25 08:48:08" runUser="eEd-2710-dev\monash" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2006">
  <TestRunConfiguration name="Local Test Run" id="eef31c20-bbef-4958-a7ce-d6984143827b">
    <Description>This is a default test run configuration for a local test run.</Description>
    <Deployment userDeploymentRoot="C:\sandRibbon\TestResults" useDefaultDeploymentRoot="false" runDeploymentRoot="C:\sandRibbon\TestResults\monash_EED-2710-DEV 2010-02-25 08_48_08">
      <DeploymentItem filename="C:\sandRibbon\SandRibbon\lib\agsXMPP.dll" />
      <DeploymentItem filename="C:\sandRibbon\SandRibbonInterop\bin\Release\SandRibbonInterop.dll" />
      <DeploymentItem filename="C:\sandRibbon\SandRibbon\bin\Debug\MeTL.exe" />
      <DeploymentItem filename="C:\sandRibbon\SandRibbon\lib\Divelements.SandRibbon.dll" />
    </Deployment>
    <TestTypeSpecific />
  </TestRunConfiguration>
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
    <RunInfos>
      <RunInfo computerName="EED-2710-DEV" outcome="Warning" timestamp="2010-02-25T08:48:09.7416543+11:00">
        <Text>Warning: Test Run deployment issue: The assembly or module 'nunit.framework' directly or indirectly referenced by the test container 'c:\sandribbon\functional\bin\release\functional.dll' was not found.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
  <Times creation="2010-02-25T08:48:08.6315908+11:00" queuing="2010-02-25T08:48:09.7456545+11:00" start="2010-02-25T08:48:10.0376712+11:00" finish="2010-02-25T08:48:14.7659417+11:00" />
  <TestDefinitions>
    <UnitTest name="BillModifiesBensConversation" storage="c:\sandribbon\functional\bin\release\functional.dll" id="90202b7c-a698-66c4-2037-feb7327acdd1">
      <Css projectStructure="" iteration="" />
      <Owners>
        <Owner name="" />
      </Owners>
      <Execution id="09fa07fa-b24f-46b0-91ff-e2650da0a4b7" />
      <TestMethod codeBase="C:\sandRibbon\Functional\bin\Release\Functional.dll" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="Functional.ConversationEditingInMultipleUserSpace, Functional" name="BillModifiesBensConversation" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="90202b7c-a698-66c4-2037-feb7327acdd1" executionId="09fa07fa-b24f-46b0-91ff-e2650da0a4b7" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="09fa07fa-b24f-46b0-91ff-e2650da0a4b7" testId="90202b7c-a698-66c4-2037-feb7327acdd1" testName="BillModifiesBensConversation" computerName="EED-2710-DEV" duration="00:00:00" startTime="2010-02-25T08:48:10.2566838+11:00" endTime="2010-02-25T08:48:14.6779366+11:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d">
      <Output>
        <ErrorInfo>
          <Message>Class Initialization method Functional.ConversationEditingInMultipleUserSpace.MyClassInitialize threw exception. System.InvalidOperationException:  System.InvalidOperationException: The target element cannot receive focus.
   at MS.Internal.Automation.ElementUtil.Invoke(AutomationPeer peer, DispatcherOperationCallback work, Object arg)
   at MS.Internal.Automation.ElementProxy.SetFocus().</Message>
          <StackTrace>    at System.Runtime.InteropServices.Marshal.ThrowExceptionForHRInternal(Int32 errorCode, IntPtr errorInfo)
   at MS.Internal.Automation.UiaCoreApi.CheckError(Int32 hr)
   at System.Windows.Automation.AutomationElement.SetFocus()
   at Functional.ConversationEditingInMultipleUserSpace.MyClassInitialize(TestContext testContext) in C:\sandRibbon\Functional\ConversationEditingInMultipleUserSpace.cs:line 45
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>